src/commands/login.ts

```diff
@@ -53,8 +56,8 @@ const formatTimeLeft = (seconds: number) => {
  * Otherwise, the identity file is written to the ~/.p0 directory.
  */
 export const login = async (
-  args: { org?: string; refresh?: boolean },
-  options?: { skipAuthenticate?: boolean }
+  args: { org: string; refresh?: boolean },
```

Comment:
I think this is intentionally left optional, users can provide the P0_ORG var or if left blank it will respond with whether or not the user is already logged in 🤔

Reply:
Thanks for finding that. Updated.